src/lib/structures/Mass.ts

```diff
@@ -169,7 +171,7 @@ This party will automatically depart in 2 minutes, or if the leader clicks the s
 	async removeUser(user: KlasaUser) {
 		if (user === this.leader) return;
 		if (!this.users.includes(user)) return false;
-		removeFromArr(this.users, user);
+		this.users = removeFromArr(this.users, user);
```

Comment:
`removeFromArr` already does mutate, so this is doing `this.users = this.users` ?

Reply:
The comments of the fn says it mutates, but it doesn't actually. I also looked up the function online as well and it says it creates a new array that is returned, and does not mutate. 

If it mutated, there wouldn't have been a bug for me to catch and fix :)